lms/resources/_js_config/__init__.py

```diff
@@ -318,9 +318,6 @@ def add_deep_linking_api(self):
                 "content_item_return_url": self._request.lti_params[
                     "content_item_return_url"
                 ],
-                "lms": {
```

Comment:
We don't need to forward this manually anymore.

I'll mention where this were used to add them to the testing instructions.

This one applies to deep linking.